plugins/explore-common/README.md

```diff
@@ -0,0 +1,3 @@
+# explore-common
+
+Shared types for the `explore*` plugins.
```

Comment:
```suggestion
Shared types for the `explore` plugin.
```

All different packages are considered part of a single plugin. Yeah it's a bit awkward but it's kinda what NPM forces on us for now 😅